DESCRIPTION

```diff
@@ -11,13 +11,24 @@ LazyData: TRUE
 Suggests:
     ElemStatLearn,
     glmnet,
-    testthat
+    testthat,
+    knitr,
+    rmarkdown
 LinkingTo: Rcpp
 Depends:
-    R (>= 2.10),
-    survival
+    R (>= 2.10)
 Imports:
     ggplot2,
+    utils,
     methods,
-    Rcpp
-RoxygenNote: 5.0.1
+    stats,
+    survival,
+    foreach,
+    doParallel,
+    data.table,
```

Comment:
I'm also curious about why you switched to `data.table` instead of the plain data frames.

Reply:
`data.table` works even when the features aren't labelled. They have better error handling with informative messages and also, they have a less memory overhead, efficient and faster for larger numbers.